app/views/world_locations/index.html.erb

```diff
@@ -19,7 +30,7 @@
     <% @world_locations.each do |type, locations| %>
       <section id="<%= type.name.pluralize.parameterize %>" class="world-location-type js-filter-block">
         <header class="type-heading">
-          <h1><%= type.name.pluralize %></h1>
+          <h2><%= type.name.pluralize %></h2>
```

Comment:
Could we use a heading component here?

Reply:
Related, but I can't comment on [line 40](https://github.com/alphagov/whitehall/pull/5788/files#diff-1d953526c50fc8bd376b4dae739632efR40).

I think it's worth updating the heading level for the letters to a level 3 heading - at the moment they're also level 2, so the hierarchy looks like this:

<img width="907" alt="" src="https://user-images.githubusercontent.com/1732331/92241422-1b6fed00-eeb6-11ea-9a79-32764a3a3c3a.png">

With a level 3 heading the letters would be nested underneath "World locations" - do you think that makes more sense?

Reply:
I'm not sure a heading component is the right choice as this element isn't styled like a heading. It's the 2 text elements on (this page)[https://www.gov.uk/world] that read "world locations" and "international delegations". They don't look like headings for design/space reasons (I assume?) behave like headings for the purpose of screen reader navigation. Do we think this not looking like a heading is problematic and if so, should this be something we change as part of this PR?

Reply:
Apologies, I meant to swap the `h2` element for a heading component:

```suggestion
        <%= render "govuk_publishing_components/components/heading", {
          text: type.name.pluralize
        } %>
```

Reply:
My comment holds up my dude. That `type.name.pluralize` element in the world page resolves twice to "world locations" and "international delegations". Maybe linking to live is a bit misleading as they're currently h1s with this PR swapping them out for h2s. Here's an example of the styling on my local:

![Screenshot 2020-09-04 at 14 30 23](https://user-images.githubusercontent.com/64783893/92244733-32650e00-eebb-11ea-9178-1176bbcaa0ac.png)

You can see that the h2 is being styled by whitehall to be tiny. The question again is do we think that these tiny h2s are problematic? I personally think they are problematic and we should style these like titles but there's the consideration of if making a design change like this is within scope. I guess we could?

Reply:
I'll make the h3 change no problem though, good spot 👍

Reply:
Ah, okaydokey - thanks for being patient. 

Could we change the CSS to target the heading directly, as it's not clear that this is a custom heading:

```scss
.world-locations {
  h2 {
```

to be something like:
```scss
.app-world-locations-heading {
```

and 
```html
<h2 class="app-world-locations-heading">...
```

Or even remove the need for a custom CSS class altogether?

```html
<h2 class="govuk-body-l">World locations</h2>
```